lldb/source/Plugins/Process/gdb-remote/ProcessGDBRemote.cpp

```diff
@@ -3656,6 +3656,12 @@ Status ProcessGDBRemote::LaunchAndConnectToDebugserver(
 
   FileSpec debugserver_path = GetDebugserverPath(*GetTarget().GetPlatform());
 
+  if (!FileSystem::Instance().Exists(debugserver_path)) {
+    return Status::FromErrorString("Could not find '" DEBUGSERVER_BASENAME
+                                   "'. Please ensure it is properly installed "
+                                   "and available in your PATH.");
```

Comment:
LLVM's coding style [says](https://llvm.org/docs/CodingStandards.html#error-and-warning-messages) that errors and warning should start without capitalization and end without a period (so they can be appended).